.github/workflows/safernet_production.yml

```diff
@@ -0,0 +1,127 @@
+# This is a basic workflow to publish a Twilio function with Github Actions
+
+name: SaferNet Brazil Production release
```

Comment:
the staging one is 'SaferNet Staging release' without the Brazil... maybe we should do 'SaferNet Production release' to keep consistent with that